platform/core/src/services/HangingProtocolService/HPMatcher.js

```diff
@@ -71,6 +71,9 @@ const match = (metadataInstance, rules, customAttributeRetrievalCallbacks) => {
       // mark that a required Rule has failed
       if (rule.required) {
         requiredFailed = true;
+        // console.log("Failed required readValues", attribute, readValues, rule, errorMessages)
```

Comment:
The console methods are really helpful when figuring out why an HP isn't applying, so I've left them commented

Reply:
So many commented console.log in this PR, please remove them all